src/main/java/com/minecolonies/coremod/entity/ai/citizen/smelter/EntityAIWorkSmelter.java

```diff
@@ -64,9 +64,56 @@ public class EntityAIWorkSmelter extends AbstractEntityAIUsesFurnace<JobSmelter,
     public EntityAIWorkSmelter(@NotNull final JobSmelter job)
     {
         super(job);
+        super.registerTargets(new AITarget(BREAK_ORES, this::breakOres, TICKS_SECOND));
         worker.setCanPickUpLoot(true);
     }
 
+    /**
+     * Break down ores until they are finished.
+     * @return the next state to go to.
+     */
+    private IAIState breakOres()
+    {
+        final ICraftingBuildingModule module = building.getFirstModuleOccurance(BuildingSmeltery.OreBreakingModule.class);
+        final IRecipeStorage currentRecipeStorage = module.getFirstFulfillableRecipe(ItemStackUtils::isEmpty, 1, false);
+
+        if(currentRecipeStorage == null)
+        {
+            return IDLE;
+        }
+
+        final ItemStack inputItem = currentRecipeStorage.getCleanedInput().stream()
+                                      .map(ItemStorage::getItemStack)
+                                      .findFirst().orElse(ItemStack.EMPTY);
+
+        if(inputItem.isEmpty())
+        {
+            return IDLE;
+        }
+
+        WorkerUtil.faceBlock(building.getPosition(), worker);
+
+        if(!module.fullFillRecipe(currentRecipeStorage))
+        {
+            return IDLE;
+        }
+        else
+        {
+            worker.decreaseSaturationForContinuousAction();
+            worker.getCitizenExperienceHandler().addExperience(0.2);
+        }
+
+        Network.getNetwork()
+          .sendToTrackingEntity(new LocalizedParticleEffectMessage(inputItem, building.getID()), worker);
+        Network.getNetwork()
+          .sendToTrackingEntity(new LocalizedParticleEffectMessage(inputItem, building.getID().below()), worker);
+
+        worker.swing(InteractionHand.MAIN_HAND);
```

Comment:
Might be useful to put at least the ore block and possibly also a pick in their hands, just for feedback purposes.  (Even if the pick is a purely imaginary one, since they don't actually use it... unless maybe they should.)

Reply:
yeah, as they dont actually use the pick, I didnt want to put anything in their hand they dont have in their inventory.

Reply:
Not sure if it should be required that they actually should use a pickaxe. Would that mean that the players would need to enchant it with Fortune I - Fortune IV for level 2 - 5? And would they need to give a diamond pickaxe to a level 1 smeltery as well (even though that is not in line with the tool material requirements for level 1)?

Reply:
No, even if we do require a pick it's easy enough to just pretend that it's enchanted, similar to how we currently do it.
But it doesn't seem like a silly idea to require a level-appropriate pick that takes durability damage as they work, as a cost of work similar to the log stripping.  (And could even be done mostly automatically with that existing code.  I think that even takes care of the tool request?  Though I'm not completely sure about that.)

Though I wasn't even really suggesting that, just something visual -- similar to how the sifter shows what they're currently working on.

Reply:
As the items currently stay in the hut inventory I guess its fine to not hold it and just show the particles. Else it would require bigger changes then.

Reply:
They stay in the hut inventory for the sifter, too.  It's just to give the player some idea of what they're doing.